ui/src/components/LoginForm/LoginForm.jsx

```diff
@@ -18,28 +21,32 @@ export default function LoginForm({handleLogin}) {
                 const res = await axios.post(`${config.API_BASE_URL}/auth/login`, {
                     "username" : username.current.value,
                     "password" : password.current.value
-                    })                
-                handleLogin(res.data.user)    
+                    })
+                handleLogin(res.data.user)
             } catch (err) {
                 alert(err)
                 console.log(err)
             }
         }
         login()
     }
-  
+
     return (
-      <form onSubmit={handleSubmit}>
-        <div className="title">Login</div>
-        <label>
-          <span>Username</span>
-          <input ref={username}></input>
-        </label>
-        <label>
-          <span>Password</span>
-          <input type="password" ref={password}></input>
-        </label>
-        <button type="submit">Login</button>
-      </form>
+      <Card className="login-card">
+        <Card.Body>
+          <Card.Title className="form-title">Login</Card.Title>
+          <Form onSubmit={handleSubmit}>
```

Comment:
I tested this and when I call with handleSubmit(), the user gets stuck on the login page. I think this is because an event gets passed in as a parameter, so handleSubmit() doesn't account for the event.

Reply:
awesome - makes sense!